.gitignore

```diff
@@ -6,62 +6,8 @@
 *vulnerabilities*report*.json
 *report*json.zip
 .version
-
+version.json
 *.code-workspace
 !project.code-workspace
-.rej
-*.porig
 
 # Please, add your custom content below!
-
-# dependencies
-node_modules
-*/node_modules
-/.pnp
-.pnp.js
-
-# testing
-/coverage
-
-# next.js
-/.next/
-/out/
-
-# production
-/build
-
-# misc
-.DS_Store
-*.pem
-
-# debug
-npm-debug.log*
-
-# local env files
-.env*.local
-.env
-
-# vercel
-.vercel
-
-# typescript
-*.tsbuildinfo
-next-env.d.ts
-
-# reports
-reports
-.reports
-
-# cypress
-tests/screenshots/*
-plugin-cache/
-
-*.terraform*
-terraform.tfstate
-terraform.tfstate.backup
-terraform/*_output.txt
-terraform/*_output.json
-group_target-env.tfvars
-
-.amplify/
-amplify_outputs.json
```

Comment:
This must not be removed from the gitignore.

Reply:
Files can be added to scripts/config/.repository-template-sync-ignore